### ISSUE TITLE
the computational cost, specifically FLOPs (Floating Point Operations)

### ISSUE THREAD

**Issue by AdKingH** [scailab/paser#1]
I'm particularly interested in understanding the computational cost, specifically FLOPs (Floating Point Operations), for your model, especially given its adaptive nature.
My question is regarding how FLOPs are calculated when the model employs different strategies for different inputs. Since the FLOPs would likely fluctuate per input due to these adaptive policies, how do you typically quantify or report the overall FLOPs for the model?

**Comment by bharats** [comment]
Hi, thanks for your question. We use the function `def get_flops(model: nn.Module, input: torch.Tensor)` in [utils.py](https://github.com/scailab/paser/blob/main/src/utils.py) to calculate the flops required for an image and a given model.

To calculate the flops of our adaptive model on our test set, we keep track of how many patches are sent to each model and then simply multiply the flop cost of each model times the number of patches sent to that model. We also account for the flops required by the global and rl models as they must process every image.
